mesonbuild/backend/xcodebackend.py

```diff
@@ -27,21 +27,22 @@ def __init__(self, build):
         self.project_conflist = self.gen_id()
         self.indent = '       '
         self.indent_level = 0
-        self.xcodetypemap = {'c' : 'sourcecode.c.c',
-                             'a' : 'archive.ar',
+        self.xcodetypemap = {'c': 'sourcecode.c.c',
+                             'a': 'archive.ar',
                              'cc': 'sourcecode.cpp.cpp',
-                             'cxx' : 'sourcecode.cpp.cpp',
-                             'cpp' : 'sourcecode.cpp.cpp',
-                             'c++' : 'sourcecode.cpp.cpp',
-                             'm' : 'sourcecode.c.objc',
-                             'mm' : 'sourcecode.cpp.objcpp',
-                             'h' : 'sourcecode.c.h',
-                             'hpp' : 'sourcecode.cpp.h',
-                             'hxx' : 'sourcecode.cpp.h',
-                             'hh' : 'sourcecode.cpp.hh',
-                             'inc' : 'sourcecode.c.h',
-                             'dylib' : 'compiled.mach-o.dylib',
-                             'o' : 'compiled.mach-o.objfile',}
+                             'cxx': 'sourcecode.cpp.cpp',
+                             'cpp': 'sourcecode.cpp.cpp',
+                             'c++': 'sourcecode.cpp.cpp',
+                             'm': 'sourcecode.c.objc',
+                             'mm': 'sourcecode.cpp.objcpp',
+                             'h': 'sourcecode.c.h',
+                             'hpp': 'sourcecode.cpp.h',
+                             'hxx': 'sourcecode.cpp.h',
+                             'hh': 'sourcecode.cpp.hh',
+                             'inc': 'sourcecode.c.h',
+                             'dylib': 'compiled.mach-o.dylib',
+                             'o': 'compiled.mach-o.objfile',
+                            }
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->